ui/src/map/layers/geometry/geometry-alignment-layer.ts

```diff
@@ -73,7 +73,7 @@ function createAlignmentFeature(
         ({ geometryId }) => geometryId == alignment.header.id,
     );
 
-    const cacheKey = `${alignment.header.id}-${resolution}-${isAlignmentSelected}`;
+    const cacheKey = `${alignment.header.id}-${resolution}-${isAlignmentSelected}-${alignment.linked}`;
```

Comment:
Kakutuksesta puuttui tieto onko ka. alignment linkitetty vai ei -> linkittyneisyyden muuttuessa näytettiin edelleen wanhaa, kakutettua tavaraa siihen saakka kunnes jonkin ulkoisen syyn (yleensä zoomailun tai refreshin) takia feature piti luoda uudestaan

Reply:
Onkohan näistä feturecacheista oikeasti enää nykyisellään hyötyä. Pisteiden määrä on nyt aika paljon pienempi kun silloin aikanaan kun näitä tehtiin. Muista layereistahan nuo taitaa olla jo pitkälti poistettu.